stable/pachyderm/values.yaml

```diff
@@ -41,22 +42,31 @@ microsoft:
 
 ## Set default image settings, resource requests and number of replicas of pachd
 pachd:
+ exposeObjApi: false
  replicaCount: 1
  ## Size of pachd's in-memory cache for PFS files. Size is specified in bytes, with allowed SI suffixes (M, K, G, Mi, Ki, Gi, etc).
  pfsCache: 0G
  ## For available images please check: https://hub.docker.com/r/pachyderm/pachd/tags/
  image:
   repository: pachyderm/pachd
-  tag: 1.7.11
+  tag: 1.8.6
   pullPolicy: Always
  worker:
   repository: pachyderm/worker
-  tag: 1.7.11
+  tag: 1.8.6
  resources:
   ## For non-local deployments, 1 cpu and 2G of memory requests are recommended
   requests:
    cpu: 250m
    memory: 512M
+  # limits:
+  #  cpu: 250m
+  #  memory: 512M
+ service:
+  grpc:
+   annotations: {}
+   port: 650
```

Comment:
We could default it to `30650`.

Reply:
done, this is the default port, I just forgot to change the port I use for my loadBalancer type :).